PennMobile/Polls/Networking/PollsNetworkManager.swift

```diff
@@ -11,85 +11,99 @@ import SwiftyJSON
 import PennMobileShared
 
 class PollsNetworkManager: NSObject, Requestable {
-
     static let id = UIDevice.current.identifierForVendor?.uuidString ?? ""
```

Comment:
Looks like GitHub ate my comment earlier, but I see two issues with this approach to anonymizing response IDs:

1. In theory the backend could trace this back to a user's access token (since the device identifier is also sent with some other requests, such as laundry). In practice, the server doesn't store this data, but it might be better to just generate a random UUID when the app first launches
2. It looks like we're sending the access token with each poll request anyway in `answerPoll`